python/lsst/ts/dimm/dimm_csc.py

```diff
@@ -161,9 +208,13 @@ async def begin_disable(self, id_data):
         self.telemetry_loop_running = False
         self.seeing_loop_running = False
 
-        self.model.controller.stop()
+        try:
+            await self.controller.stop()
+        except Exception:
+            self.log.exception("Error in begin_disable. Continuing...")
 
         await super().begin_disable(id_data)
+        self.cmd_disable.ack_in_progress(id_data, timeout=60)
```

Comment:
You may want to do this first thing in the method.

Reply:
Done